deploy/rhsm-clowdapp.yaml

```diff
@@ -247,9 +237,6 @@ objects:
       - replicas: 3
         partitions: 3
         topicName: platform.rhsm-subscriptions.tasks
-      - replicas: 3
-        partitions: 3
-        topicName: platform.rhsm-subscriptions.openshift-metering-tasks
```

Comment:
It looks like this file missed the memo about the topicName dropping the word "openshift" from the name, so when I moved this chunk to the swatch-metrics/deploy/clowdapp.yaml file, I updated it then.